spec/board_spec.rb

```diff
@@ -28,4 +28,26 @@
       end
     end
   end
+
+  describe "#next_token" do
```

Comment:
Is `next_token` used outside of the `Board` class?

Reply:
It isn't.  I initially took out the spec, but then I put it back in.  In the future I might need to use next_token from outside Board, and wanted to indicate that it's a stable method.

Reply:
Do you think I should take out the spec?

Reply:
Yeah, I would recommend taking it out of the spec and explicitly making it private. That will make it easier for us to change if we need to. If/when we need to make it public we can.

Reply:
Sounds good.  Made the change.